docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -4,12 +4,11 @@
 [id="hermetic-builds_{context}"]
 = Enabling hermetic builds
 
-Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This enhanced dependency visibility allows the build system to capture and produce a more precise record of the build's origin, dependencies, and modifications than it would otherwise be possible.
+A hermetic build is a secure, self-contained build process that doesn't depend on anything outside of the build environment. This means that it does not have network access, is not vulnerable to external influences, and cannot fetch dependencies at run time. Instead, you must declare all required resources and dependencies in your build definition. 
 
-In {ProductName}, hermetic builds are achieved by blocking network access to the build process. Therefore, you must pre-fetch all dependencies before the build can start.
-
-IMPORTANT: Hermetic builds disable network access, so the builds that require dependencies outside its Git repository could fail.
+In {ProductName}, you can run a hermetic build by blocking network access to the build process. This means that you must fetch all dependencies _before_ the build can start.
```

Comment:
>  you can run a hermetic build by blocking network access

A user cannot block it manually. It can be done via the parameter. I'd rephrase.

Reply:
Like [here](https://github.com/redhat-appstudio/image-controller/blob/main/.tekton/image-controller-push.yaml#L34-L35).